uber/site_sections/preregistration.py

```diff
@@ -517,7 +495,6 @@ def confirm(self, session, message='', return_to='confirm', undoing_extra='', **
 
                 page = ('badge_updated?id=' + attendee.id + '&') if return_to == 'confirm' else (return_to + '?')
                 if attendee.amount_unpaid:
-                    cherrypy.session['return_to'] = page
```

Comment:
IIRC, previously we would direct attendees to the group management page when they were done registering, if they had claimed a badge in the group. We now explicitly _don't_ want this because the group management page allows you to change dealer applications. So I've ripped a lot of this stuff out in favor of always directing someone either back to the donation form (if the charge class returned an error) or to their confirmation page.